app/assets/javascripts/discourse/tests/acceptance/user-preferences-sidebar-test.js

```diff
@@ -0,0 +1,214 @@
+import { test } from "qunit";
+
+import { click, visit } from "@ember/test-helpers";
+
+import {
+  acceptance,
+  exists,
+  updateCurrentUser,
+} from "discourse/tests/helpers/qunit-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+acceptance("User Preferences - Sidebar - Tagging Disabled", function (needs) {
+  needs.settings({
+    tagging_enabled: false,
+  });
+
+  needs.user({
+    experimental_sidebar_enabled: true,
+    sidebar_category_ids: [],
+    sidebar_tag_names: [],
+  });
+
+  test("user should not see tag chooser", async function (assert) {
+    await visit("/u/eviltrout/preferences/sidebar");
+
+    assert.ok(!exists(".tag-chooser"), "tag chooser is not displayed");
+  });
+});
+
+acceptance("User Preferences - Sidebar - Tagging Enabled", function (needs) {
```

Comment:
This contains test cases for adding categories to the sidebar (line 64 and 109), but the title here feels specific to tags. Should the title here be broadened? Or move the categories tests to another `acceptance` block?

Reply:
I removed `Tagging Enabled` in this case as it can be implied from the tests. I want to be able to share the setup for the tests within the same acceptance block.